src/vivarium_census_prl_synth_pop/components/businesses.py

```diff
@@ -194,18 +194,18 @@ def on_time_step(self, event: Event):
             )
 
         # change jobs if of working age already
-        working_age = pop.loc[pop.age >= WORKING_AGE].index
-        changing_jobs = self.randomness.filter_for_rate(
-            working_age,
-            self.job_change_rate(working_age)
-            * event.step_size.days
-            / utilities.DAYS_PER_YEAR,
+        working_age_idx = pop.loc[pop.age >= WORKING_AGE].index
+        changing_jobs_idx = filter_by_rate(
```

Comment:
I'm personally a fan of "_idx" is you also have, say, boolean masks floating around